### PR TITLE
Update Readme For Linux Installation

### PR DESCRIPTION
**Pull request by Rupinderwadali**

### CODE REVIEW

**Comment by BaaniLeen** [comment]
@Rupinderwadali Here is a small change to maintain the uniformity of the README section. Make the Database Setup as a separate heading without the counter '3.' and place '3.' as a counter for the point below :)
The rest, @AkankshaBodhankar can confirm :)

**Comment by Rupinderwadali** [comment]
Sure @BaaniLeen. Will made the changes if required. @AkankshaBodhankar Please check and let me know if the changes required.
Thanks :+1:

File: README.md [hunk 1]
```diff
@@ -82,6 +82,23 @@ Or download the ZIP file from above
...
+Or download the ZIP file from above
+
+
+### 3. Database Setup
+Go to 'http://localhost/phpmyadmin', in the top menu bar of the phpMyAdmin panel, click on Import. Now, click on 'Choose the file'. Select the firstaide_web.sql file present in database folder of this project and import it. Check the database credentials in dbconnect.php file.
+
+4. Open the http://localhost/FirstAide-web/
+
+5. Setting up the reCaptcha as per steps mentioned in Installation steps for Windows.
+
```
Comment (fatimarafiqui) [inline README.md]: We are working on distributed documentation for installation. This will be covered during the GSOC'17 cycle.

File: login.php [hunk 2]
```diff
@@ -59,7 +59,7 @@
       $password = mysqli_real_escape_string($connection, $_POST['password']);
       $email = stripslashes($email);
       $password = stripslashes($password);
-      $password = md5($email.$password);
+      //$password = md5($email.$password);
```
Comment (fatimarafiqui) [inline login.php]: Never remove password encryption in any case.

**Comment by Rupinderwadali** [comment]
@fatimarafiqui the project was not showing the result, I asked about this in group where they suggest me this change. Is there any other alternative for running the project?

**Comment by fatimarafiqui** [comment]
@Rupinderwadali I did not get your query. Can you please elaborate?

**Comment by fatimarafiqui** [comment]
@Rupinderwadali Distributed documentation is one of the agendas of GSoC'17. This will be covered.
@AkankshaBodhankar @MehaKaushik I think we should close this PR.

**Comment by MehaKaushik** [comment]
Due to inactivity and this being a part of GSoC'17, we have closed this issue.

**Pull request closed without merging**